### PR TITLE
WIP: Datarow Metadata Beta

### PR DESCRIPTION
**Pull request by gdj0nes**
SDK code for the beta of datarow metadata

### CODE REVIEW

File: labelbox/orm/model.py [hunk 1]
```diff
@@ -90,6 +91,10 @@ def Enum(enum_cls: type, *args):
     def Json(*args):
         return Field(Field.Type.Json, *args)
 
+    @staticmethod
+    def List(*args):
+        return Field(Field.Type.List, *args)
+
```
Comment (msokoloff1) [inline labelbox/orm/model.py:97]: I don't think this is being used

File: tests/integration/test_data_row_metadata.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+def test_get_datarow_metadata_ontology(dr_md_ontology):
+    mdo = dr_md_ontology
+    assert len(mdo.all_fields)
+    assert len(mdo.reserved_fields)
+    assert len(mdo.custom_fields) == 0
+
+
+def test_get_datarow_metadata(datarow):
+    """No metadata"""
+    md = datarow.make_metadata
```
Comment (msokoloff1) [inline tests/integration/test_data_row_metadata.py:76]: This function doesn't exist. Is this something that should be added?

File: labelbox/schema/data_row.py [hunk 3]
```diff
@@ -63,13 +90,6 @@ def bulk_delete(data_rows):
         """
         BulkDeletable._bulk_delete(data_rows, True)
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.metadata.supports_filtering = False
-        self.metadata.supports_sorting = False
-        self.attachments.supports_filtering = False
-        self.attachments.supports_sorting = False
```
Comment (msokoloff1) [inline labelbox/schema/data_row.py:71]: Why did you remove this?

File: labelbox/schema/data_row_metadata.py [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+                    DataRowMetadataSchema(**option)
+                    for option in schema["options"]
+                ]
+
+            schema["options"] = options
+            fields.append(DataRowMetadataSchema(**schema))
+
+        return fields
+
+    def bulk_upsert(
```
Comment (msokoloff1) [inline labelbox/schema/data_row_metadata.py:186]: Why is the ontology object responsible for uploading metadata?

**Comment by msokoloff1** [comment]
I feel like some of this functionality could be moved to the DataRow object. Also this is totally ignoring the sdk magic which I am fine with but maybe can chat about it some more.

**Comment by manuaero** [comment]
Let's do the metadata SDK in a proper way. It's a super core feature
